src/drivers/gitlab.js

```diff
@@ -176,8 +186,7 @@ class Gitlab {
     let { url } = opts;
 
     if (endpoint) {
-      this.api_v4 = this.api_v4 || (await this.detect_api_v4());
-      url = `${this.api_v4}${endpoint}`;
+      url = `${await this.detect_api()}/api/${API_VER}${endpoint}`;
```

Comment:
Here we're awaiting a function from the templated string itself, but the `comment_create` and `upload` functions above use an intermediate variable. Would it make any sense to standardize on a common approach?

Reply:
I don't choose one over the other just only if I need more space or not typing.
If the line is short enough I use this over using an extra var.

Reply:
I mean what prevalences is the line length not the variable assignment

Reply:
👌